Code/Framework/AzFramework/AzFramework/Process/ProcessCommunicatorTracePrinter.cpp

```diff
@@ -19,6 +19,10 @@ ProcessCommunicatorTracePrinter::ProcessCommunicatorTracePrinter(AzFramework::Pr
 ProcessCommunicatorTracePrinter::~ProcessCommunicatorTracePrinter()
 {
     // flush stdout
+    Pump();  // get any remaining data if available and split it by newlines
+    
+    // if there's any further data left over in the buffer then make sure it gets written, too
+    
```

Comment:
remove newline

Reply:
thanks for noticing these (And the ones below.  I'll go ahead and fix it as part of submit.